beta2-quickstart-master/counter-contract/tests/harness.rs

```diff
@@ -35,13 +35,13 @@ async fn get_contract_instance() -> (MyContract, ContractId) {
 // ANCHOR: contract-test
 #[tokio::test]
 async fn can_get_contract_id() {
-    let (instance, _id) = get_contract_instance().await;
+    let (_instance, _id) = get_contract_instance().await;
```

Comment:
Underscores are used for unused variables in Rust, so I think we should keep this as-is.